client/src/components/Icons/SVGIcon.css

```diff
@@ -0,0 +1,20 @@
+.logoNav {
+	background-color: #fae5dd;
+	list-style: none;
+	color: black;
+	display: flex;
+	flex-direction: column;
+	justify-content: space-around;
+	padding: 20px;
+	margin: 10px;
+}
+
+.theIcon {
+	display: flex;
+	flex-direction: column;
+	justify-content: space-around;
+}
+
+.icon {
+	padding: 10px;
+}
```

Comment:
Can you think of more semantic names for these? It's going to be really challenging to remember the difference between `.theIcon` and `.icon`